dom/Cargo.toml

```diff
@@ -1,10 +1,13 @@
 [package]
-name = "dom"
-version = "0.0.1"
+name = "stylo_dom"
+version = "0.1.0"
 authors = ["The Servo Project Developers"]
+documentation = "https://docs.rs/stylo_dom/"
+description = "Support crate for Stylo CSS engine"
```

Comment:
Maybe mention the DOM, it's a bit weird to have such a generic description.

Reply:
Yeah. I guess so. This crate barely even has anything to do with the DOM though. It's just the case that it contains types that are defined within Gecko's DOM module (these types previously lived within `style_traits` which makes more sense from a Servo perspective).

Reply:
Updated to mention to the DOM.